docs/samples/Microsoft.ML.Samples/Dynamic/BinaryClassification/AveragedPerceptron.cs

```diff
@@ -0,0 +1,61 @@
+﻿using Microsoft.ML;
+
+namespace Microsoft.ML.Samples.Dynamic.BinaryClassification
+{
+    public static class AveragedPerceptron
+    {
+        public static void Example()
+        {
+            // In this examples we will use the adult income dataset. The goal is to predict
```

Comment:
Can you move this to `<Summary>` and `<Remarks>` on `Example`?

Reply:
IMO, let's stay away from xml documentation inside samples. I don't see any of that in .Net samples, and that is harder to read that plain text. 

---
In reply to: [255265822](https://github.com/dotnet/machinelearning/pull/2483#discussion_r255265822) [](ancestors = 255265822)

Reply:
I checked with John. For tutorials they don't use any xml comments. Just regular // comments. we can follow the same template.

[https://github.com/dotnet/samples/blob/master/machine-learning/tutorials/SentimentAnalysis/Program.cs](https://github.com/dotnet/samples/blob/master/machine-learning/tutorials/SentimentAnalysis/Program.cs)


---
In reply to: [255266200](https://github.com/dotnet/machinelearning/pull/2483#discussion_r255266200) [](ancestors = 255266200,255265822)